### PR TITLE
[ci]: Bump rust, mold and nightly to 1.60, 1.2.0 and 1.62 respectively.

### PR DESCRIPTION
**Pull request by appetrosyan**
Signed-off-by: Aleksandr Petrosyan <a-p-petrosyan@yandex.ru>
### Description of the Change

Version bump. 

### Issue

None

### Benefits

Update

### Possible Drawbacks

None

### CODE REVIEW

**Review by mversic — APPROVED** [review]

File: .github/workflows/iroha2.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ jobs:
         with:
           context: .
           push: true
-          tags: hyperledger/iroha2:2.0.0-pre-rc.2
+          tags: hyperledger/iroha2:current
```
Comment (s8sato) [inline .github/workflows/iroha2.yml:53]: Distinguish from `latest`?

**Review by s8sato — DISMISSED** [review]

File: .github/workflows/iroha2.yml (hunk 1 above)
Comment (mversic) [reply .github/workflows/iroha2.yml:53]: `latest` sounds more idiomatic to me

Comment (appetrosyan) [reply .github/workflows/iroha2.yml:53]: Latest means latest, as in the one that we are working on. 
Current means, the currently recommended version. 
Perhaps `stable`? But then we have `LTS`

Comment (mversic) [reply .github/workflows/iroha2.yml:53]: I think latest should refer to latest published release. What we're working on should be tagged with `-dev` or alike

Comment (mversic) [reply .github/workflows/iroha2.yml:53]: I thought this was the convention. If, I'm wrong feel free to override me

Comment (mversic) [reply .github/workflows/iroha2.yml:53]: `latest` tag semantics are overloaded so it should be avoided. I would prefer if you used `dev`

Comment (appetrosyan) [reply .github/workflows/iroha2.yml:53]: Ah, I see. This is not for the latest development container, rather for the latest release.

Comment (mversic) [reply .github/workflows/iroha2.yml:53]: ok, so why the switch from the semantic versioning? If it's the latest release it should have a proper version assigned

**Review by Arjentix — APPROVED** [review]

**Review by mversic — APPROVED** [review]

**Pull request merged**